Project/Project/CDTTodoReplicator.m

```diff
@@ -1,203 +0,0 @@
-    CDTReplicatorFactory *factory = delegate.replicatorFactory;
-    CDTPushReplication *replicationConfig = [CDTPushReplication replicationWithSource:delegate.datastore
-                                                                               target:url];
-    NSError *error;
-    CDTReplicator *aPushReplicator = [factory oneWay:replicationConfig error:&error];
-    aPushReplicator.sessionConfigDelegate = self;
-    if (!aPushReplicator) {
-        [self log:@"Error creating push replicator: %@", error];
-    }
-    else {
-        [self startAndFollowReplicator:aPushReplicator label:@"push" taskGroup:taskGroup];
-    }
-}
-/**
- Starts a replication and sets self as the delegate
- 
- In real apps, you'd probably use the replicatorDidComplete: and replicatorDidError:
- callbacks to do something useful, updating the UI or showing an error for example.
- */
--(void)startAndFollowReplicator:(CDTReplicator*)replicator label:(NSString*)label taskGroup:(dispatch_group_t)taskGroup{
-
-    NSString *state = [CDTReplicator stringForReplicatorState:replicator.state];
-    [self log:@"%@ state: %@ (%d)", label, state, replicator.state];
-
-    [replicator setDelegate:self];
-
-    NSError *error;
-    if (![replicator startWithTaskGroup:taskGroup error:&error]) {
-        [self log:@"error starting %@ replicator: %@", label, error];
-        state = [CDTReplicator stringForReplicatorState:replicator.state];
-        [self log:@"%@ state: %@ (%d)", label, state, replicator.state];
-        return;
-    }
-
-    @synchronized(self){
-        [self.replicators addObject:replicator];
-    }
-}
-
--(void)log:(NSString*)format, ... {
-    va_list args;
-    va_start(args, format);
-    NSString *message = [[NSString alloc] initWithFormat:format arguments:args];
-    va_end(args);
-
-    NSLog(@"%@", message);
-}
-
-
-#pragma mark CDTReplicatorDelegate
-
--(void)replicatorDidComplete:(CDTReplicator *)replicator
-{
-    
-    [self log:@"%@ complete", replicator];
-    @synchronized(self){
-        [self.replicators removeObject:replicator];
-    }
-}
-
--(void)replicatorDidError:(CDTReplicator *)replicator info:(NSError *)info
-{
-    
-    [self log:@"%@ error: %@", replicator, info];
-    @synchronized(self) {
-        [self.replicators removeObject:replicator];
-    }
-}
-
--(void)replicatorDidChangeState:(CDTReplicator *)replicator
-{
-
-    NSString *state = [CDTReplicator stringForReplicatorState:replicator.state];
-    [self log:@"%@ new state: %@ (%d)", replicator, state, replicator.state];
-}
-
--(void)replicatorDidChangeProgress:(CDTReplicator *)replicator
-{
-
-    NSString *state = [CDTReplicator stringForReplicatorState:replicator.state];
-    [self log:@"%@ changes total %ld. changes processed %ld. state: %@ (%d)", replicator,
-     replicator.changesTotal, replicator.changesProcessed, state, replicator.state];
-}
-
-- (void)customiseNSURLSessionConfiguration:(nonnull NSURLSessionConfiguration *)config {
```

Comment:
I'm wondering if it is wise to include both ways of replicating the sample application, it could lead to confusion. Perhaps what we should do is remove the reference to the sample project, with the aim that the sample project is a quick start, rather than show off all the advanced features such as polices.